app/src/organisms/PipetteWizardFlows/AttachProbe.tsx

```diff
@@ -50,46 +85,73 @@ export const AttachProbe = (props: AttachProbeProps): JSX.Element | null => {
   } = props
   const { t, i18n } = useTranslation('pipette_wizard_flows')
   const pipetteWizardStep = { mount, flowType, section: SECTIONS.ATTACH_PROBE }
+  const [isPending, setIsPending] = React.useState<boolean>(false)
+  const [showUnableToDetect, setShowUnableToDetect] = React.useState<boolean>(
+    false
+  )
+  const [numberOfTryAgains, setNumberOfTryAgains] = React.useState<number>(0)
+
   const pipetteId = attachedPipettes[mount]?.serialNumber
   const displayName = attachedPipettes[mount]?.displayName
   const is8Channel = attachedPipettes[mount]?.data.channels === 8
   const is96Channel = attachedPipettes[mount]?.data.channels === 96
   const calSlotNum = 'C2'
   const axes: MotorAxes = mount === LEFT ? ['leftZ'] : ['rightZ']
+  const { refetch, data: attachedInstrumentsData } = useInstrumentsQuery({
+    enabled: false,
+    onSettled: () => {
+      setIsPending(false)
+    },
+  })
+  const attachedPipette = attachedInstrumentsData?.data.find(
+    (instrument): instrument is PipetteData =>
+      instrument.ok && instrument.mount === mount
+  )
 
   if (pipetteId == null) return null
   const handleOnClick = (): void => {
-    chainRunCommands?.(
-      [
-        {
-          commandType: 'home' as const,
-          params: {
-            axes: axes,
-          },
-        },
-        {
-          commandType: 'home' as const,
-          params: {
-            skipIfMountPositionOk: mount,
-          },
-        },
-        {
-          commandType: 'calibration/calibratePipette' as const,
-          params: {
-            mount: mount,
-          },
-        },
-        {
-          commandType: 'calibration/moveToMaintenancePosition' as const,
-          params: {
-            mount: mount,
-          },
-        },
-      ],
-      false
-    )
+    setIsPending(true)
+    refetch()
       .then(() => {
-        proceed()
+        if (attachedPipette?.state?.tipDetected === true) {
```

Comment:
shouldn't this work?

```suggestion
        if (attachedPipette?.state?.tipDetected) {
```

Reply:
yeah it does! I think I was trying to get rid of a lint warning but it isn't necessary